test/extended/util/framework.go

```diff
@@ -106,6 +106,24 @@ func WaitForBuilderAccount(c kclient.ServiceAccountsInterface) error {
 	return wait.Poll(60, time.Duration(1*time.Second), waitFunc)
 }
 
+// WaitForDeployerAccount waits until the deployer service account gets fully
+// provisioned
+func WaitForDeployerAccount(c kclient.ServiceAccountsInterface) error {
+	waitFunc := func() (bool, error) {
+		sc, err := c.Get("deployer")
+		if err != nil {
+			return false, err
+		}
+		for _, s := range sc.Secrets {
+			if strings.Contains(s.Name, "dockercfg") {
```

Comment:
The deployer needs a non-empty list of `ImagePullSecrets`, as opposed to mountable secret containing that value.